# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lirc/smp-safe.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- lirc-0.8.0pre4/drivers/lirc_parallel/lirc_parallel.c.orig   2005-09-22 22:10:37.000000000 +0200
+++ lirc-0.8.0pre4/drivers/lirc_parallel/lirc_parallel.c        2006-01-20 11:41:59.000000000 +0100
@@ -43,7 +43,7 @@
 #include <linux/config.h>

 #ifdef CONFIG_SMP
-#error "--- Sorry, this driver is not SMP safe. ---"
+#warning "--- Sorry, this driver is not SMP safe. ---"
 #endif

 #include <linux/module.h>
